crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -587,3 +616,11 @@ sp_api::decl_runtime_apis! {
         fn chain_constants() -> ChainConstants;
     }
 }
+
+#[cfg(all(feature = "std", feature = "runtime-benchmarks"))]
+fn kzg_instance() -> &'static Kzg {
```

Comment:
This is an expensive function. To get accurate benchmarks, do we need to call this function before we run the benchmarks? Or does the benchmark harness already pre-warm or discard outliers?

Reply:
Benchmark does multiple runs and first call alone will initialize it and rest of the calls from the same benchmark wont. The exec time will be saturated by the end of the benchmark.

I have validated these by running more than our usual `benchmarking.steps` to much bigger value. Did not see a big difference but rather some small pico seconds